vello_tests/tests/property.rs

```diff
@@ -65,8 +65,8 @@ fn empty_scene(use_cpu: bool) {
     assert_eq!(image.format, Format::Rgba8);
     for pixel in image.data.data().chunks_exact(4) {
         let &[r, g, b, a] = pixel else { unreachable!() };
-        let image_color = Color::rgba8(r, g, b, a);
-        if image_color != color {
+        let image_color = Color::from_rgba8(r, g, b, a);
+        if image_color.premultiply().difference(color.premultiply()) > 1e-4 {
```

Comment:
Why this premultiply step?

Reply:
There's no `AlphaColor::difference` because of perceptual difference vs alpha.

Reply:
(I've previously pasted this code to Raph with no objections.)